### DISCUSSION TITLE
Saved Variants Download Table with FXNL Data

### DISCUSSION THREAD

**Discussion by dspeck1** [broadinstitute/seqr#2122] [Q&A]
We have a local instance of Seqr installed in GKE based on the latest code in this repo.   When downloading Saved Variants to CSV for Tier 2 - Novel Gene and Phenotype variants we do not see FXNL data as a column.  Is this supposed to be included or do we need to modify code to bring VariantFunctionalData into columns returned in sample_data_api.py or other part of code?  Thanks!

**Comment by hanars** [comment]
Functional data is currently not included in the seqr variant download. Adding it would require a code change. If you decide to implement this, please feel free to let us know/ put up a PR! You can also submit a "feature request" github ticket for our team to implement this, but it will probably be a while before we have the bandwidth to work on this
